addons/ui/org.openhab.ui.cometvisu/META-INF/MANIFEST.MF

```diff
@@ -12,6 +12,7 @@ Import-Package:
  javax.servlet,
  javax.servlet.annotation,
  javax.servlet.http,
+ javax.sql,
```

Comment:
Hm, not sure, everything seemed to work with these changes. The bundle started, cometvisu was accessible. There was no message/error at all about another missing import.